### DISCUSSION TITLE
Add access to fields from collection in coditions

### DISCUSSION THREAD

**Discussion by NickSettler** [directus/directus#7660] [Feature Requests]
I suppose it would be great, if will be added access to the fields in the colletion when setting conditions property of the field. 

### Example
**Meetings collection**
| field | type |
|-|-|
| id | integer |
| name | string |
| people | M2M connection |
| goods  | O2M connection |

**People collection**
| field | type |
|-|-|
| id | integer |
| name | string |
| meetings | M2M connection -> `Meetings` collection |

**Goods collection**
| field | type |
|-|-|
| id | integer |
| name | string |
| meeting | M2O connection -> `Meetings` collection |
| people_who_used | M2M connection -> `People` collection |

### Explanation

I create a meeting in the `meeting`. Some people from `people` collection will come to this meeting. Each of them will bring some goods from `goods` collection. And each good will be used by any person in this meeting. 

### Idea

It would be greatful, if there will be an option to create a condition for the `goods` field in the `meetings` collection. The condition will allow to choose `people_who_used` of the `goods` only from people who mentioned in the `people` field `meetings` collection item.

**Comment by rijkvanzanten** [comment]
> The condition will allow to choose people_who_used of the goods only from people who mentioned in the people field meetings collection item.

I believe we're very close to supporting this 🙂 The conditions already support overriding the interface options, and https://github.com/directus/directus/pull/7080 adds support to configure filters used in the relational interfaces, which combined should allow for what you mentioned above 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
